exercises/scale-generator/description.md

```diff
@@ -32,6 +30,19 @@ Note that by common music theory convention the natural notes "C" and "a"
 follow the sharps scale when ascending and the flats scale when descending. 
 For the scope of this exercise the scale is only ascending.
 
+### Task
+
+Given a tonic, generate the 12 note musical scale starting with the tonic.
+
+- Shift the base scale appropriately so that all 12 notes are returned
+starting with the given tonic.
+- For the given tonic, determine if the scale is to be returned with flats
+or sharps.
+- Return the major scale (i.e. uppercase notes) irrespective of if the
+given tonic is a major or minor note (i.e. uppercase or lowercase respectively).
```

Comment:
being that there are `Interval()` tests with lowercase input and it seems to me the lack of such tests for `Chromatic()` is an oversight (or maybe as you point out the existence of such tests at all is the oversight) so lowercase needs to be accounted for. When you get to the `Interval()` task the only additional logic you should need to account for is the interval, `Chromatic()` should just be a special case of `Interval()` with a specific interval pattern so I think it makes sense to mention it here.

That said I can update the language to avoid the major/minor terminology (i clearly don't know anything about it LOL).